tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapAsyncOpsTest.java

```diff
@@ -69,77 +57,76 @@ public void setup(TestContext testContext) throws Exception {
-                            count.destroyCount.incrementAndGet();
-                            break;
-                    }
-                } catch (DistributedObjectDestroyedException ignored) {
-                }
-            }
-        }
-    }
-
     @Verify(global = true)
     public void globalVerify() throws Exception {
         IList<MapOperationsCount> results = targetInstance.getList(basename + "report");
-        MapOperationsCount total = new MapOperationsCount();
-        for (MapOperationsCount i : results) {
-            total.add(i);
+        MapOperationsCount totalMapOperationsCount = new MapOperationsCount();
+        for (MapOperationsCount mapOperationsCount : results) {
+            totalMapOperationsCount.add(mapOperationsCount);
         }
-        log.info(basename + ": " + total + " total of " + results.size());
+        log.info(basename + ": " + totalMapOperationsCount + " total of " + results.size());
     }
 
     @Verify(global = false)
     public void verify() throws Exception {
         Thread.sleep(maxTTLExpirySeconds * 2);
 
-        final IMap map = targetInstance.getMap(basename);
+        IMap map = targetInstance.getMap(basename);
         log.info(basename + ": map size  =" + map.size());
     }
 
+    @RunWithWorker
+    public AbstractWorkerTask<Operation> createBaseWorker() {
+        return new WorkerTask();
+    }
+
+    private class WorkerTask extends AbstractWorkerTask<Operation> {
+        private final IMap<Integer, Object> map = targetInstance.getMap(basename);
+
+        public WorkerTask() {
+            super(operationSelectorBuilder);
+        }
+
+        @Override
+        protected void doRun(Operation operation) {
+            try {
+                int key = nextInt(keyCount);
+                switch (operation) {
+                    case PUT_ASYNC:
+                        Object value = nextInt();
+                        map.putAsync(key, value);
+                        count.putAsyncCount.incrementAndGet();
+                        break;
+                    case PUT_ASYNC_TTL:
+                        value = nextInt();
+                        int delay = 1 + nextInt(maxTTLExpirySeconds);
+                        map.putAsync(key, value, delay, TimeUnit.SECONDS);
+                        count.putAsyncTTLCount.incrementAndGet();
+                        break;
+                    case GET_ASYNC:
+                        map.getAsync(key);
+                        count.getAsyncCount.incrementAndGet();
+                        break;
+                    case REMOVE_ASYNC:
+                        map.removeAsync(key);
+                        count.removeAsyncCount.incrementAndGet();
+                        break;
+                    case DESTROY:
+                        map.destroy();
+                        count.destroyCount.incrementAndGet();
+                        break;
+                }
+            } catch (DistributedObjectDestroyedException ignored) {
```

Comment:
DO we want this? If you get this exception, shit must be happening imho and should not be caught.